docs-src/descriptions/one-source.jade

```diff
@@ -714,6 +714,58 @@ div
 
 
 
++descr-method('buffer-with-time-or-count', 'bufferWithTimeOrCount', 'obs.bufferWithTimeOrCount(interval, count, [options])').
+  Continuously buffers values from the source observable, flushing
+  every #[b interval] milliseconds, or immediately once #[b count]
+  values have been stored.
+  Also flushes the buffer before end, but you can disable that
+  by passing #[tt {flushOnEnd: false}] as #[b options].
+  
+p.
+  Limited by time:
+
+pre.javascript(title='example')
+  :escapehtml
+    var source = Kefir.sequentially(100, [1, 2, 3, 4, 5, 6]);
+    var result = source.bufferWithTimeOrCount(310, 10);
+    result.log();
+
+pre(title='console output')
+  :escapehtml
+    > [sequentially.bufferWithTimeOrCount] <value> [1, 2, 3]
+    > [sequentially.bufferWithTimeOrCount] <value> [4, 5, 6]
+    > [sequentially.bufferWithTimeOrCount] <end>
+
+pre(title='events in time').
+  source:  ---1---2---3---4---5---6X
+  result:  -----------•-----------•X
+                [1,2,3]     [4,5,6]
```

Comment:
We should move `result`'s points a bit to show that flushes happen not at the same time with events in `source`, also the second point is the "flush on end" and should be shown a bit differentially. Let's modify it like this:

```js
var source = Kefir.sequentially(100, [1, 2, 3, 4, 5, 6, 7, 8]);
var result = source.bufferWithTimeOrCount(330, 10);
```
```
  source:  ---1---2---3---4---5---6---7---8 X
  result:  ------------•------------•------•X
                 [1,2,3]      [4,5,6]  [7,8]
```